velox/vector/BaseVector.cpp

```diff
@@ -502,6 +502,9 @@ std::string BaseVector::toString(bool recursive) const {
 }
 
 std::string BaseVector::toString(vector_size_t index) const {
+  VELOX_CHECK(
```

Comment:
Use VELOX_CHECK_LT for better error message.

Do not use fmt::format to avoid generating a string in happy case.